### PR TITLE
Add ModelMesh playground

### PR DESCRIPTION
**Pull request by gz65555**
- Add ModelMesh playground

### CODE REVIEW

File: demos/model-mesh/index.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+cameraEntity.transform.setPosition(0, 10000, 10000);
+cameraEntity.transform.lookAt(new Vector3(0, 8000, 0));
+const camera = cameraEntity.addComponent(Camera);
+camera.farClipPlane = 400000;
+camera.fieldOfView = 55;
+
+// createCube(engine, rootEntity);
+createPlane(engine, rootEntity);
+engine.run();
+Logger.enable();
```
Comment (GuoLei1990) [inline demos/model-mesh/index.ts:23]: This can delete

**Review by zhuxudong — CHANGES_REQUESTED** [review]
Unfortunately, demosify does not currently support multiple files. so you should run `npm run add-demo` and combine these files into one file. you can run `npm run dev:demosify` to preview effect online

File: demos/model-mesh/plane.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    mesh.uploadData(false);
+    this.count = count;
+  }
+}
+
+export function createPlane(engine: Engine, entity: Entity) {
+  const planeEntity = entity.createChild("plane");
+  planeEntity.transform.setRotation(-90, 0, 0);
+  const meshRenderer = planeEntity.addComponent(MeshRenderer);
+  const plane = PrimitiveMesh.createPlane(engine, 1245000, 1245000, 100, 100, false);
```
Comment (GuoLei1990) [inline demos/model-mesh/plane.ts:93]: The unit is too large, usually one unit is assumed to be one meter.
Maybe Causes floating point precision problems.

File: demos/model-mesh/plane.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  planeEntity.addComponent(PlaneAnimation);
+  
+  engine.resourceManager
+    .load<Texture2D>({
+      url: "https://gw.alipayobjects.com/mdn/rms_2e421e/afts/img/A*fRtNTKrsq3YAAAAAAAAAAAAAARQnAQ",
+      type: AssetType.Texture2D
+    })
+    .then((res) => {
+      const mtl = new Material(engine, shader);
+      mtl.shaderData.setTexture("u_baseColor", res);
```
Comment (GuoLei1990) [inline demos/model-mesh/plane.ts:104]: cace shaderData as a const

File: demos/model-mesh/plane.ts [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  const plane = PrimitiveMesh.createPlane(engine, 1245000, 1245000, 100, 100, false);
+  meshRenderer.mesh = plane;
+  planeEntity.addComponent(PlaneAnimation);
+  
+  engine.resourceManager
+    .load<Texture2D>({
+      url: "https://gw.alipayobjects.com/mdn/rms_2e421e/afts/img/A*fRtNTKrsq3YAAAAAAAAAAAAAARQnAQ",
+      type: AssetType.Texture2D
+    })
+    .then((res) => {
```
Comment (GuoLei1990) [inline demos/model-mesh/plane.ts:102]: `res` use `texture` is better

File: demos/model-mesh/plane.ts (hunk 2 above)
Comment (gz65555) [reply demos/model-mesh/plane.ts:93]: +1

File: demos/model-mesh/index.ts (hunk 1 above)
Comment (gz65555) [reply demos/model-mesh/index.ts:23]: +1

File: demos/model-mesh/plane.ts (hunk 4 above)
Comment (gz65555) [reply demos/model-mesh/plane.ts:102]: +1

File: demos/model-mesh/plane.ts (hunk 3 above)
Comment (gz65555) [reply demos/model-mesh/plane.ts:104]: I can't understand..

**Pull request merged**